Frontend/CO.CDP.OrganisationApp/Pages/OneLogin.cshtml.cs

```diff
@@ -66,10 +68,21 @@ private async Task<IActionResult> UserInfo()
                 session.Set(Session.UserDetailsKey, ud);
             }
 
+            var personInviteId = session.Get<Guid?>("PersonInviteId");
```

Comment:
I see you have added session on invitation landing page,

Reply:
Ideally this should have been done via redirect url

Reply:
I saw an opportunity to save a lot of extra work and potential code duplication by claiming the person invite here. Do you think there is much extra benefit doing it elsewhere?

Reply:
I don't think so that there shouldn't be any extraordinary amount of work is involved.
The Onelogin page should only be involved with login process and any necessary redirects after that, nothing else.